app/views/brexit_checker/_action_list.html.erb

```diff
@@ -1,7 +1,7 @@
 <% items.each.with_index do | action, index | %>
   <% action_index = "#{audience_index}.#{'%02d' % (index + 1)}"%>
-  <div class="govuk-grid-row govuk-!-margin-bottom-3">
-    <div class="govuk-grid-column-three-quarters">
+  <div class="govuk-grid-row brexit-checker__action">
+    <div class="govuk-grid-column-full brexit-checker__content-wrapper">
```

Comment:
Is `govuk-grid-column-full` needed?

Reply:
Interesting, to remove it seems to widen the page, but it looks better!
Good catch

Reply:
also checked mobile, looks good